components/content-picker/PickedItem.tsx

```diff
@@ -62,13 +76,16 @@ const PickedItem = ({ item, isOrderable, handleItemDelete, mode, id }) => {
 	// empty, it will return null, which is handled in the effect below.
 	const preparedItem = useSelect(
 		(select) => {
-			const { getEntityRecord, hasFinishedResolution } = select('core');
+			// @ts-ignore
+			const { getEntityRecord, hasFinishedResolution } = select(coreStore);
 
 			const getEntityRecordParameters = [type, item.type, item.id];
-			const result = getEntityRecord(...getEntityRecordParameters);
+			const result: any = getEntityRecord(...getEntityRecordParameters);
```

Comment:
This is erroring on me `TS2556: A spread argument must either have a tuple type or be passed to a rest parameter.`